testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -21,7 +21,15 @@ Feature: Smoke tests for <client>
 
   Scenario: Client <client> grains are displayed correctly on the details page
     Given I am on the Systems overview page of this "<client>"
-    Then I can see all system information for "<client>"
+    Then the hostname for "<client>" should be correct
+    And the kernel for "<client>" should be correct
+    And the OS version for "<client>" should be correct
+    And the IPv4 address for "<client>" should be correct
+#    And the IPv6 address for "<client>" should be correct
```

Comment:
I guess the same reason as the other commented out, bsc1193858 please add a comment here as well so we remember to uncomment when the bug is fixed/closed

Reply:
Fixed.